infrastructure/http_metadata_service.go

```diff
@@ -87,12 +87,25 @@ func (ms HTTPMetadataService) GetPublicKey() (string, error) {
 		return "", nil
 	}
 
+	url := fmt.Sprintf("%s%s", ms.metadataHost, ms.sshKeysPath)
+	r, err := ms.GetHttpContent(url)
```

Comment:
is there something else we could check on? ie empty body, status code, etc?

Reply:
Hi @cppforlife Thanks for your suggestion and I will improve it.